src/util/types.ts

```diff
@@ -1198,6 +1198,11 @@ export interface LabelLayoutOption {
      * @default 'none'
      */
     hideOverlap?: boolean
+    /**
+     * If labels overlap. It will Remove overlaps;
+     * @default 'none'
+     */
+    removeOverlap?: boolean
```

Comment:
`moveOverlap` is used for different label layout algorithms. Perhaps we can add a value to it to represent this default layout algorithm.
```ts
// Apply default layout algorithm. Which will be this one I think.
moveOverlap: true
// Or name of this specific layout algorithm.
moverOverlap: 'xxx'
```